server/src/internalClusterTest/java/org/opensearch/action/IndicesRequestIT.java

```diff
@@ -565,8 +572,9 @@ public void testPutMapping() {
     }
 
     public void testGetSettings() {
-        interceptTransportActions(GetSettingsAction.NAME);
 
+        interceptTransportActions(GetSettingsAction.NAME);
+        stubClusterTermResponse(internalCluster().getClusterManagerName());
```

Comment:
why do we need stub response for integ tests?

Reply:
Since there is different transport call to get the ClusterTerm version, it required to stub the fake response of returning mismatched term version.

Reply:
can this be UT/ functional test and not integ test.

Reply:
Moved this test to a seperate UT and removed stubbing from IT